docs/Way-of-Working/UbiquitousLanguage.md

```diff
@@ -0,0 +1,227 @@
+# Ubiquitous Language
+
+For clarity and consistency, this document defines key terms used within our team across various domains (e.g., Data
+Science, IT).
+
+The goal is to provide a list of commonly used terms and terms that are good to know. It is not to a full reference of
+all existing terms.
+
+Terms can and have been used 'as is' from sources mentioned at the end of this document.
+
+## Terms used in Data Science
+
+### Algorithm
+
+An algorithm is a sequence of repeatable steps, often expressed mathematically, written by a human and executed by a
+computer, to solve a certain type of data science problem. The main concept is that an algorithm takes in some input and
+produces an output, and the same input will always produce the same output. In machine learning, algorithms take in
+input in the form of data and [hyperparameters](https://www.datacamp.com/blog/data-science-glossary#hyperparameter),
+identify and learn common patterns from the data, and produce outputs in the form of predictions.
+
+### Bias
+
+Bias refers to the tendency of models to underfit the data, leading to inaccurate predictions in machine learning and
+data science. Moreover, bias can also mean algorithmic bias, which refers to the propensity of machine learning models
+to reproduce harmful societal biases by treating different groups of individuals differently based on protected
+attributes such as race, sexual orientation, gender identify, age, pregnancy and more.
+
```

Comment:
I don't think this is entirely correct. Bias and underfitting are conflated.

```suggestion
Bias refers to the tendency of a model to favor certain outcomes over others, potentially leading to inaccuracy or unfairness.

Moreover, bias can also mean algorithmic bias, which refers to the propensity of machine learning models
to reproduce harmful societal biases by treating different groups of individuals differently based on protected
attributes such as race, sexual orientation, gender identify, age, pregnancy and more.
```

Reply:
Bias can mean many different things and often very precise things, for example [statistical bias](https://en.wikipedia.org/wiki/Bias_of_an_estimator). I would not mention these definitions here, but just focus on algorithmic bias to avoid confusion.